Projects/MouseControl/mouse_control_movement.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+"""
```

Comment:
Also, this could use an atexit() definition to stop the robot if it crashes or is stopped wile the motors are moving.